src/map/pc.c

```diff
@@ -5662,6 +5668,8 @@ int pc_setpos(struct map_session_data* sd, unsigned short map_index, int x, int
 		if (sd->npc_id)
 			npc->event_dequeue(sd);
 		npc->script_event(sd, NPCE_LOGOUT);
+		
+		clan->member_offline(sd);
```

Comment:
This should be handled in `unit_free`.
Search for this comment -
`// Notify friends that this char logged out. [Skotlex]` and add it below.